spark-metrics-template.yaml

```diff
@@ -103,22 +120,28 @@ objects:
             image: ${SPARK_IMAGE}
             env:
               - name: SPARK_METRICS_ON
-                value: "true"
+                value: ${SPARK_METRICS_ON}
               - name: SPARK_MASTER_ADDRESS
                 value: spark://${MASTER_NAME}:7077
               - name: SPARK_MASTER_UI_ADDRESS
                 value: http://${MASTER_NAME}-webui:8080
+              - name: SPARK_USER
+                value: admin
 parameters:
 - name: SPARK_IMAGE
   description: Name of the Spark master/worker image
-  value: analyticsci/openshift-spark:v2.1.0-metrics-jolokia
+  value: analyticsci/openshift-spark:v2.1.0-metrics-prometheus
```

Comment:
this should change to just a analyticsci/openshift-spark:v2.2.0 value I think, since metrics will be carried in all of our images.  In fact, why not radanalytics.io/openshift-spark:2.2-latest?

Reply:
We can do that